sdk/monitor/Microsoft.OpenTelemetry.Exporter.AzureMonitor/src/TelemetryPartA.cs

```diff
@@ -49,6 +52,29 @@ internal static TelemetryItem GetTelemetryItem(Activity activity, string instrum
             return telemetryItem;
         }
 
+        internal static TelemetryItem GetTelemetryItem(LogRecord logRecord, string instrumentationKey)
+        {
+            var name = PartA_Name_Mapping[TelemetryType.Message];
+            var time = logRecord.Timestamp.ToString(CultureInfo.InvariantCulture);
+
+            TelemetryItem telemetryItem = new TelemetryItem(name, time)
+            {
+                InstrumentationKey = instrumentationKey
+            };
+
+            // TODO: I WAS TOLD THIS MIGHT BE CHANGING. IGNORING FOR NOW.
+            //InitRoleInfo(activity);
+            //telemetryItem.Tags[ContextTagKeys.AiCloudRole.ToString()] = RoleName;
+            //telemetryItem.Tags[ContextTagKeys.AiCloudRoleInstance.ToString()] = RoleInstance;
+
+            telemetryItem.Tags[ContextTagKeys.AiOperationId.ToString()] = logRecord.TraceId.ToHexString();
```

Comment:
Do this inside `if (logRecord.TraceId != default)`

Reply:
should i do the same for the SpanId?

Reply:
n/m just saw your other comment :)